### PR TITLE
boards/limifrog-v1: fix uart configuration in periph_conf

### PR DESCRIPTION
**Pull request by PeterKietzmann**
Must have been missed during restruction of the stm32l1 uart driver

### CODE REVIEW

File: boards/limifrog-v1/include/periph_conf.h [hunk 1]
```diff
@@ -76,8 +76,8 @@ static const timer_conf_t timer_config[] = {
 /* UART 0 pin configuration */
 #define UART_0_PORT         GPIOC
```
Comment (thomaseichinger) [inline boards/limifrog-v1/include/periph_conf.h]: @PeterKietzmann  Could you remove these while you are at it?

**Comment by katezilla** [comment]
Ack if travis agrees :-)

File: boards/limifrog-v1/include/periph_conf.h [hunk 2]
```diff
@@ -76,8 +76,8 @@ static const timer_conf_t timer_config[] = {
...
 #define UART_0_PORT_CLKEN() (RCC->AHBENR |= RCC_AHBENR_GPIOCEN)
-#define UART_0_RX_PIN       11
-#define UART_0_TX_PIN       10
+#define UART_0_RX_PIN       GPIO_PIN(PORT_C, 11)
+#define UART_0_TX_PIN       GPIO_PIN(PORT_C, 10)
 #define UART_0_AF           7
```
Comment (thomaseichinger) [inline boards/limifrog-v1/include/periph_conf.h]: `#define UART_0_AF  GPIO_AF7`

**Comment by thomaseichinger** [comment]
@PeterKietzmann Please compare `boards/nucleo-l1/include/periph_conf.h` in general.

**Comment by PeterKietzmann** [comment]
@thomaseichinger @katezilla I adapted some other configurations. Won't do more for now.

**Comment by thomaseichinger** [comment]
@PeterKietzmann Great, thanks a lot! @katezilla Could you test this?

**Comment by katezilla** [comment]
yes, tested it, works, fixed the uart bug.

**Comment by thomaseichinger** [comment]
@katezilla so this is an implicit ACK?

**Comment by katezilla** [comment]
yes, still ACK

**Comment by PeterKietzmann** [comment]
So why no one hits the merge button?! Ang go

**Pull request merged**